Setup.hs

```diff
@@ -4,4 +4,5 @@ import Distribution.Simple
 
 main :: IO ()
 main = defaultMainWithHooks myHooks
-  where myHooks = simpleUserHooks
+  where
+    myHooks = simpleUserHooks
```

Comment:
Why did you change this?

Reply:
This happened when `tools/run-formatter.sh` was used for formatting.

Reply:
It was run on all of the the repository at some point.